libcontainer/container_linux.go

```diff
@@ -518,6 +515,17 @@ func (c *Container) shouldSendMountSources() bool {
 		return false
 	}
 
+	return true
+}
+
+// shouldSendMountSources says whether the child process must setup bind mounts with
+// the source pre-opened (O_PATH) in the host user namespace.
+// See https://github.com/opencontainers/runc/issues/2484
+func (c *Container) shouldSendMountSources() bool {
+	if !c.shouldSend() {
+		return false
```

Comment:
This seems pointless, just `return c.shouldSend()` would do it. But no point in having two functions either (`shouldSendMountSources` and `shouldSendIdmapSources`) if they both do exactly the same.

This is breaking the detection on when to send each :(

Reply:
Ohh, I see what I was missing. Ignore this, sorry